src/clientManager.js

```diff
@@ -18,7 +18,7 @@ ClientManager.prototype.init = function(identHandler, sockets) {
 	this.identHandler = identHandler;
 	this.webPush = new WebPush();
 
-	if (!Helper.config.public) {
+	if (!Helper.config.public && !Helper.config.ldap.enable) {
```

Comment:
Ah right, forgot to add comment for this. Reason is because of removing this https://github.com/thelounge/lounge/pull/1443/files#diff-e23e6a2fa48b03eef4ac89b3a4656f5aL29

So I figured LDAP just takes care of user management. Am I incorrect? Happy to remove, LDAP stuff is a black box to me anyway.